core/templates/dev/head/pages/exploration-editor-page/preview-tab/preview-tab.directive.html

```diff
@@ -36,7 +36,7 @@
     position: fixed;
     right: 0;
     padding-bottom: 2em;
-    z-index: 1060;
+    z-index: 1020;
```

Comment:
What do you think about deleting padding-bottom and also increasing the "bottom" property here, so that there's no overlap?

Reply:
![image](https://user-images.githubusercontent.com/11008603/60401091-1f113100-9b9a-11e9-9878-5986b6cf9060.png)

Yes, that allows the hint button to be clickable. But if you look closely it looks like the popover is hidden completely below the "Restart" button.

Reply:
![image](https://user-images.githubusercontent.com/11008603/60401137-caba8100-9b9a-11e9-9747-da5d48217af5.png)

It should look like this with the z-index fix.